conf/solr/crawldb/conf/managed-schema

```diff
@@ -116,20 +116,48 @@
 
+   <!-- Deprecated Fields -->
+   <!-- <field name="jobId" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" /> -->
+   <!-- <field name="lastFetchedAt" type="date" indexed="true" stored="true" multiValued="false" docValues="true" /> -->
+   <!-- <field name="numTries" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="numFetches" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="status" type="status" indexed="true" stored="true" default="NEW" multiValued="false" required="true" docValues="true" /> -->
+   <!-- <field name="depth" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="plainText" type="text_general" indexed="true" stored="true" multiValued="false" /> -->
+
+   <!-- Proposed Sparkler Schema -->
+   <field name="fetch_timestamp" type="date" indexed="true" stored="true" multiValued="false" docValues="true" /> 
+   <field name="content_type" type="string" indexed="true" stored="true" multiValued="false" docValues="true" /> 
+   <field name="crawler" type="string" indexed="true" stored="true" multiValued="false" default="sparkler"/> 
+   <field name="crawl_id" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" /> 
+   <field name="raw_content" type="text_general" indexed="true" stored="true" multiValued="false" />
+   <field name="status" type="status" indexed="true" stored="true" default="UNFETCHED" multiValued="false" required="true" docValues="true" />
+   <field name="extracted_text" type="text_general" indexed="true" stored="true" multiValued="false" />
+   <field name="fetch_status_code" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
+   <field name="hostname" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" />
+   <field name="signature" type="string" indexed="true" stored="true" multiValued="false" docValues="true" />
+   <field name="version" type="float" indexed="true" stored="true" required="true" default="1.0" multiValued="false" docValues="true" />
+   <field name="outlinks" type="string" indexed="true" stored="true" multiValued="true" />
+   <field name="crawler_discover_depth" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
+   <field name="crawler_fetch_depth" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
+   <field name="relative_path" type="string" indexed="true" stored="true" multiValued="false" docValues="true" />
+   <field name="parent" type="string" indexed="true" stored="true" multiValued="false" docValues="true" />
+   <field name="inlinks" type="string" indexed="true" stored="true" multiValued="true" />
```

Comment:
@thammegowda Thanks for the details! However, there is no intention to store/update the inlinks while crawling. There was a suggestion to use this field in the post crawl utilities like graph creation. 
Therefore, this field is left for future use and no related code has been added in Sparkler.

_It takes less time to clarify than to assume something wrong and explain!_